tests/ZfrOAuth2Test/Server/Grant/AuthorizationGrantTest.php

```diff
@@ -244,7 +255,7 @@ public function testCanCreateTokenResponse($hasRefreshGrant)
 
         $response = $this->grant->createTokenResponse($request, new Client(), $owner);
 
-        $body = json_decode($response->getContent(), true);
+        $body = json_decode($response->getBody(), true);
```

Comment:
I know that Guzzle has done similarly for v3, and will likely have some for the upcoming PSR-7-based v4 as well. There are a few common ones I think folks will want: strings, iterators/generarators, JSON (pass it a structure, it casts to a JSON representation), etc.